.travis.yml

```diff
@@ -11,9 +12,8 @@ env:
 services:
   - memcached
 before_install:
-  - sudo apt-get install swig
-  - wget https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.3.2.deb && sudo dpkg -i elasticsearch-1.3.2.deb
-  - sudo /usr/share/elasticsearch/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
+  - wget https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.3.2.tar.gz && tar xvf elasticsearch-1.3.2.tar.gz
+  - elasticsearch-1.3.2/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
 install:
   - pip install tox
 before_script:
```

Comment:
It's much faster on the Marketplace.  You can see https://travis-ci.org/mozilla/zamboni/builds/46229240 for the latest numbers on my PR.  It takes 35-40s to run the pip command now.  Before I added the caching, eg. https://travis-ci.org/mozilla/zamboni/builds/46118204 , it took around 130-150s to run.